src/main/kotlin/lotto/model/WinningCounter.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.model
+
+import lotto.model.LottoPlace.FIRST
+import lotto.model.LottoPlace.SECOND
+import lotto.model.LottoPlace.THIRD
+import lotto.model.LottoPlace.FOURTH
+
+data class WinningCounter(val counter: MutableMap<LottoPlace, Int> = mutableMapOf(FIRST to 0, SECOND to 0, THIRD to 0, FOURTH to 0)) : Map<LottoPlace, Int> by counter {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
data class WinningCounter(
    val counter: MutableMap<LottoPlace, Int> = mutableMapOf(
        FIRST to 0,
        SECOND to 0,
        THIRD to 0,
        FOURTH to 0
    )
) : Map<LottoPlace, Int> by counter {
```